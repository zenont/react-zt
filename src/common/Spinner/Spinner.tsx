import React, { Props, StatelessComponent } from 'react'

export const Spinner: StatelessComponent<Props<HTMLDivElement>> = ({ children }) => (
	<div className="zt-component-spinner">
		{children}
	</div>
)

export default Spinner
