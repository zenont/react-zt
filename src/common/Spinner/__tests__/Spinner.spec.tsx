import React from 'react'
import { shallow } from 'enzyme'
import { Spinner } from '../Spinner'

describe('<Spinner />', () => {
	it('should render', () => {
		const wrapped = shallow(<Spinner />)
		expect(wrapped).toMatchSnapshot()
	})
})
