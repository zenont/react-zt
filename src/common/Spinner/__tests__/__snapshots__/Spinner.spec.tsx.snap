// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render 1`] = `
<div
  className="zt-component-spinner"
/>
`;
